apps/mailrelay2/mailrelay/image-policy.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: image.toolkit.fluxcd.io/v1alpha2
```

Comment:
Do we want to automatically update mail relay?

Reply:
Seems to be already done for one of them https://github.com/hmcts/sds-flux-config/pull/1374/files

Reply:
Just checked, it does seem to be automated in prod.